src/harness/testcases/testdata/wdb_1/WDB_1_PAL_Database.csv

```diff
@@ -0,0 +1,2 @@
+palId,licensee,ownerId,registrationInformation:frn,registrationInformation:registrationDate,license:licenseId,license:licenseDate,license:licenseExpiration,license:licenseAreaIdentifier,license:licenseAreaExtent,license:licenseFrequencyBlockId,LicenseStatus,channelAssignment:primaryAssignment:lowFrequency,channelAssignment:primaryAssignment:highFrequency,channelAssignment:secondaryAssignment:highFrequency,channelAssignment:secondaryAssignment:lowFrequency

```

Comment:
This should not be here.

Reply:
this file is removed as suggested before.